rps.js

```diff
@@ -0,0 +1,25 @@
+const playGame = function (userInput) {
+    let randomNum = Math.floor(Math.random() * Math.floor(3));
+    let computerInput = ""
+    if (randomNum == 0) {
+        computerInput += "rock";
+
+    } else if (randomNum == 1) {
+        computerInput += "paper";
+    } else {
+        computerInput += "scissors"
```

Comment:
An array would be a great idea to use here as well. Think of how you might use one instead of if conditions.